inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
This PR does not fix disabled tests in this [PR](https://github.com/openvinotoolkit/openvino/pull/2921)?

Reply:
@ilyachur should I enable them? I'm not sure why they were disabled - there is no description in PR. CI passed when I was introducing my serialization changes, so I don't know how this is related to my change.

Reply:
These tests failed sporadically in CI. @ababushk can provide more details

Reply:
There's JIRA issue number https://github.com/openvinotoolkit/openvino/blob/c8dd831fc3ba68a256ab47edb4f6bf3cb5e804be/inference-engine/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp#L37
But it includes only logs to builds that were failed in CI.

There is no good way to check sporadic (aka flaky) tests except trying to re-run them few (let's say 10) times in Jenkins. You can enable these tests in your PR and I'll help you to re-run CI jobs.

Reply:
Ok. I read CVS-42029. For sure I didn't address this problem in this PR. I propose to address it in separate PR - we can add some debug logs to better identify root-cause and then re-run CI few times as suggested to reproduce. I can take the ticket on me.

Reply:
Thank you @jdanieck , this works for me